lib/main.dart

```diff
@@ -261,6 +268,7 @@ class _HomeScreenState extends State<HomeScreen> {
   }
 }
 
+// Carlo
```

Comment:
Good practice for responsive design to have WideLayout and NarrowLayout classes to provide different layouts for different screen sizes.